src/LanguageServer/Protocol/Handler/Completion/CompletionResultFactory.cs

```diff
@@ -293,6 +305,100 @@ void PromoteCommonCommitCharactersOntoList()
             }
         }
 
+        private static async Task<LSP.CompletionItem> CreateItemAndPopulateTextEditAsync(
+            Document document,
+            SourceText documentText,
+            bool supportsVSExtensions,
+            bool snippetsSupported,
+            bool itemDefaultsSupported,
+            TextSpan defaultSpan,
+            string typedText,
+            CompletionItem item,
+            CompletionService completionService,
+            CancellationToken cancellationToken)
+        {
+            if (supportsVSExtensions)
+            {
+                return await CreateVsItemAndPopulateTextEditAsync(
+                    document,
+                    documentText,
+                    snippetsSupported,
+                    itemDefaultsSupported,
+                    defaultSpan,
+                    item,
+                    completionService,
+                    cancellationToken).ConfigureAwait(false);
+            }
+
+            var lspItem = new LSP.CompletionItem() { Label = item.GetEntireDisplayText() };
+
+            if (item.IsComplexTextEdit)
+            {
+                //await completionService.GetChangeAsync(document, item, cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
is this supposed to be commented out?  Might be historical but I think it can be deleted.

Reply:
Yeah, this was commented out before my change. It was copied from here: https://github.com/dotnet/roslyn/blob/80d8d7321dd56ba3e0af5d9e9f706bfd38ba03ed/src/LanguageServer/Protocol/Handler/Completion/DefaultLspCompletionResultCreationService.cs#L44-L51.

I'll remove this.